### ISSUE TITLE
[Accessibility][Progress Bar][Sample CSS] Background colors of the progress bar have insufficient color contrast

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1727]
**Steps to Reproduce**
1. Compare background colors between completed and remaining sections of progress bar with a color contrast tool.

**Current Behavior**
The completed portion of the progress bar is blue (#337AB7) while the remaining portion is gray (#747474). This gives a color contrast ratio of 1.03:1. As a result, it is difficult for some users to perceive the difference between the completed and remaining sections of the progress bar. Although the percentages are visible, users with low vision or who cannot perceive color may be unable to determine the progress at a glance.

**Expected behavior/code**
UI elements that convey information provide at least 3:1 color contrast.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Users with limited vision and/or limited color perception

**WCAG SC**
1.4.11 Non-Text Contrast

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES
